src/library/GenerateNominations/index.tsx

```diff
@@ -194,10 +194,13 @@ export const GenerateNominations = ({
   };
 
   const disabledMaxNominations = () => {
-    return nominations.length >= maxNominations;
+    return maxNominations.isLessThanOrEqualTo(nominations.length);
   };
   const disabledAddFavorites = () => {
-    return !favoritesList?.length || nominations.length >= maxNominations;
+    return (
+      !favoritesList?.length ||
+      maxNominations.isLessThanOrEqualTo(nominations.length)
```

Comment:
This is different logic to what was there before. You've switched the numbers around so it should be `isLessThan`.

Reply:
why miss the equalTo?

Reply:
because a >= b != b <= a.

a >= b == b < a

Reply:
Sorry for the rookie mistakes and thank you for the patience